Doxygen Build Error with 1.14.0: Failing to Find '@copydoc' Targets

CI has been failing on main since #6e2097e29270bf6dcaf3e971cbe6ce8d3d59e697, which is first commit where the runners were using Doxygen 1.14.0. The commit right before it was using Doxygen 1.13.2.

These are the exact failures from the log:
```
/Users/runner/work/ice/ice/cpp/include/Ice/NativePropertiesAdmin.h:35: error: @copybrief or @copydoc target 'PropertiesAdmin::setProperties' not found
/Users/runner/work/ice/ice/cpp/include/Ice/NativePropertiesAdmin.h:32: error: @copybrief or @copydoc target 'PropertiesAdmin::getPropertiesForPrefix' not found
/Users/runner/work/ice/ice/cpp/include/Ice/NativePropertiesAdmin.h:29: error: @copybrief or @copydoc target 'PropertiesAdmin::getProperty' not found
```

Notes:
1) `PropertiesAdmin` (and the referenced functions) do exist, but they're generated code.
2) I checked the docs generated with 1.13.2, and the `copydoc` tags were working correctly there. So it's not like this has always been broken, and Doxygen just added better validation or something.
3) Doxygen can find the file just fine. Changing a link from `@copydoc PropertiesAdmin::setProperties` to `@copydoc PropertiesAdmin::dispatch` results in doxygen finding the comment just fine. It's something specific to these functions.
4) Also, if you look at the current docs: https://code.zeroc.com/ice/main/api/cpp/namespaceIce.html, Doxygen doesn't seem to generate documentation for the generated `PropertiesAdmin`.

--- a/cpp/doxygen/Doxyfile
+++ b/cpp/doxygen/Doxyfile
@@ -1041,8 +1041,6 @@ EXCLUDE_SYMBOLS        = DataStormI \
                          Ice::LoggerAdminPtr \
                          Ice::Process \
                          Ice::ProcessPtr \
-                         Ice::PropertiesAdmin \
-                         Ice::PropertiesAdminPtr \
                          IceMX::MetricsAdmin \
                          IceMX::MetricsAdminPtr \
                          IceBox::ServiceManager \

--- a/cpp/include/Ice/NativePropertiesAdmin.h
+++ b/cpp/include/Ice/NativePropertiesAdmin.h
@@ -26,13 +26,10 @@ namespace Ice
 
         ~NativePropertiesAdmin() final;
 
-        /// @copydoc PropertiesAdmin::getProperty
         std::string getProperty(std::string key, const Current& current) final;
 
-        /// @copydoc PropertiesAdmin::getPropertiesForPrefix
         PropertyDict getPropertiesForPrefix(std::string prefix, const Current& current) final;
 
-        /// @copydoc PropertiesAdmin::setProperties
         void setProperties(PropertyDict newProperties, const Current& current) final;
 
         /// Registers an update callback that will be invoked when a property update occurs.

--- a/csharp/src/Ice/DocGen.cs
+++ b/csharp/src/Ice/DocGen.cs
@@ -28,13 +28,3 @@ public partial interface Process
 public abstract partial class ProcessDisp_
 {
 }
-
-[EditorBrowsable(EditorBrowsableState.Never)]
-public partial interface PropertiesAdmin
-{
-}
-
-[EditorBrowsable(EditorBrowsableState.Never)]
-public abstract partial class PropertiesAdminDisp_
-{
-}

